README.md

```diff
@@ -49,6 +49,8 @@ cd /{ base_folder }/{{ cookiecutter.repo_slug }}
 make
 ```
 
+![alt text](./docs/images/installation.gif)
```

Comment:
Please, change the "alt text" with something meaningful like "quickstart"

Reply:
Done.